src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceWriter.cs

```diff
@@ -40,41 +37,39 @@ private void WriteInternal()
             WriteFileHeader();
 
             WriteLine("namespace Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax");
-            WriteLine("{");
+            OpenBlock();
             WriteLine();
             this.WriteGreenTypes();
             this.WriteGreenVisitors();
             this.WriteGreenRewriter();
             this.WriteContextualGreenFactories();
             this.WriteStaticGreenFactories();
-            WriteLine("}");
+            CloseBlock();
         }
 
         private void WriteSyntax()
         {
-            WriteFileHeader();
-
+            WriteLine("// <auto-generated />");
             WriteLine();
+
             WriteLine("namespace Microsoft.CodeAnalysis.CSharp.Syntax");
-            WriteLine("{");
+            OpenBlock();
             WriteLine();
             this.WriteRedTypes();
-            WriteLine("}");
+            CloseBlock();
         }
 
         private void WriteMain()
         {
             WriteFileHeader();
 
-            WriteLine();
```

Comment:
Many WriteLines were removed or moved to make newlines more consistent between constructs.